tests/e2e-openshift/monitoring/check_metrics.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+SECRET=$(oc get secret -n openshift-user-workload-monitoring | grep prometheus-user-workload-token | head -n 1 | awk '{print $1}')
+TOKEN=$(echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o json | jq -r '.data.token') | base64 -d)
+THANOS_QUERIER_HOST=$(oc get route thanos-querier -n openshift-monitoring -o json | jq -r '.spec.host')
+
+metrics="tempostack_status_condition tempooperator_build_info tempo_distributor_bytes_received_total tempo_distributor_traces_per_batch_bucket tempo_distributor_spans_received_total tempo_ingester_bytes_received_total tempo_ingester_live_traces tempo_ring_members tempo_query_frontend_connected_clients"  # Space-separated list of metrics
```

Comment:
I'm not sure if it makes sense to check every single metric. If Tempo maintainers add/remove/modify any of the metrics, this test will fail. I think it is enough to check we are getting metrics (without taking into account the name nor the value of the metric itself)

Reply:
I have modified the script to check the metrics used in the prometheus rules when monitoring is enabled for TempoStack. I found that some metrics are not available and filed a bug. This will help us to detect any regression of any of the metrics changes or are removed, which are used for alerting.